Training dataset size: 400
Training dataset size: 400
  0%|                                                  | 0/1000 [00:00<?, ?it/s]
Loss after 00004 batches: 0.956
Loss after 00008 batches: 0.923
Loss after 00012 batches: 0.896
Loss after 00016 batches: 0.881
Loss after 00020 batches: 0.873
  0%|                                                  | 0/1000 [00:57<?, ?it/s]
Traceback (most recent call last):
  File "/home/kebl6872/REFUGE_4YP/Run/train_neat.py", line 222, in model_pipeline
    train(model,train_loader,test_loader,criterion,eval_criterion,config)
  File "/home/kebl6872/REFUGE_4YP/Run/train_neat.py", line 120, in train
    cup_loss,disk_loss = test(model,test_loader,eval_criterion,config,best_valid_score,batch_ct)
  File "/home/kebl6872/REFUGE_4YP/Run/train_neat.py", line 86, in test
    data_str = f"Valid score improved from {best_valid_score[0]:2.8f} to {val_score:2.8f}. Saving checkpoint: {checkpoint_path_lowloss}"
NameError: name 'checkpoint_path_lowloss' is not defined
model tested on 400 imagesval_score: 0.19260277308437754 f1_scores [0.98337448 0.38520555 0.         0.46765566]